cmd/meroxa/root/apps/deploy.go

```diff
@@ -127,6 +128,8 @@ func (d *Deploy) createApplication(ctx context.Context) error {
 	input := meroxa.CreateApplicationInput{
 		Name:     appName,
 		Language: d.lang,
+		GitSha:   "hardcoded",
+		Pipeline: meroxa.EntityIdentifier{Name: null.StringFrom("default")},
```

Comment:
I'll work on connecting the pipeline created by Turbine immediately after [this other PR](https://github.com/meroxa/turbine/pull/16) ships.

Reply:
it's started here https://github.com/meroxa/cli/pull/276/files#diff-9660bb3708bfbd2d8adc60d0b68dfd003cbf13421ea4212747439a8c11fc54b5R168